deploy.gradle

```diff
@@ -12,4 +12,14 @@ task deployGoldfingerRx(dependsOn: [
 ]) {
     group = "Deploy"
     description = "Deploy module to repositories"
+}
+
+task deployDebug(dependsOn: [
```

Comment:
Could you name this deployLocal because debug is misleading here?

Reply:
Yes, done